finance-portal/templates/configmap-auth-init.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.config.is_populate.name | quote }}
+  labels:
+    app: {{ template "finance-portal.fullname" . }}
```

Comment:
These labels are not following best practices --> https://helm.sh/docs/topics/chart_best_practices/labels/

See the following example: https://github.com/mojaloop/helm/blob/master/centralledger/chart-service/templates/deployment.yaml#L6

Please update.